app/src/Components/DisplayCards/DisplayCards.tsx

```diff
@@ -1,9 +1,8 @@
-import { useState, useEffect } from "react";
+import { useState, useEffect, useRef } from "react";
 import ProductCard from "../ProductCard/ProductCard";
 import './DisplayCards.css';
 import ReactPaginate from "react-paginate";
 import { PokemonCard } from "../../../../api/src/interfaces/cards.interface";
-import { set } from "mongoose";
 
 interface Props {
   search?: string,
```

Comment:
I think you can remove these `?` symbols and always expect these values to be strings.

Reply:
I think the ? means the value does not need to be provided.

Reply:
Oh wait I misunderstood your comment. You are correct, the ? need to be removed.